label_sync/repos.yaml

```diff
@@ -0,0 +1,6 @@
+org: kubernetes
+repos:
+- full_name: kubernetes/test-infra
+  name: test-infra
+- full_name: kubernetes/community
+  name: community
```

Comment:
This file (when used as a parameter --repos=repos.yaml) limits list of repositories to run `sync_labels` on.